docker-compose/docker-compose.yaml

```diff
@@ -144,12 +192,6 @@ services:
       timeout: 10s
       retries: 5
       start_period: 300s
-    depends_on:
```

Comment:
Did we bump our minimum docker-compose version requirement? This `depends_on` I believe had to be removed because `service_completed_successfully` was unavailable in our old compose version.

Reply:
Right, we can't use `depends_on`: with a `condition: service_completed_successfully` without upgrading docker-compose. But we can keep the existing `condition: service_healthy` entries

Reply:
Doh, I was looking at the conversation view and though we were talking about a different part of the changes. I'll add these back in now. 🤦‍♀️